Lib/test/test_logging.py

```diff
@@ -3901,6 +3901,20 @@ def test_handlers(self):
         self.assertIs(handlers[2].formatter, f)
         self.assertIs(handlers[0].formatter, handlers[1].formatter)
 
+    def test_force(self):
+        with support.captured_stdout() as output:
+            logging.basicConfig(stream=sys.stdout)
+            logging.warning('warn')
+            logging.info('info')
+            self.assertEqual(len(logging.root.handlers), 1)
+            logging.basicConfig(level=logging.INFO, stream=sys.stdout,
+                                force=True)
+            logging.info('info')
+            sys.stdout.seek(0)
```

Comment:
Is this `seek()` call needed? Doesn't `getvalue()` get the entire output, anyway?